project/LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2010-2016 Thomas Rabaix
+Copyright (c) 2010-2018 Thomas Rabaix
```

Comment:
I think I read somewhere that you can just put "…2010 Thomas Rabaix" and be done with it forever

Reply:
o use twig to generate a date

Reply:
https://github.com/symfony/form/commit/973a310098c590abe3e35a90ed2a7389f26158b3

was overridden with:

https://github.com/symfony/form/commit/0d89adcacf5be46c572d074302a0e0aa52e25fee

So I guess Symfony updates it every year, should we?

Reply:
@greg0ire I saw that too but never find something that confirms it.